pytorch_module_path: "/home/demo/Workspace/Assembly-Part-Segmentation/furniture_part_segmentation"
# config_file: "filled_rgb_late_fusion_freeze_small.json"
config_file: "filled_rgb_late_fusion_freeze_small.json"
weight_path: "/home/demo/catkin_ws/src/assembly_part_recognition/pretrained_weights/furniture_part/0715_rgb_albumentation/epoch_6.tar"
width: 640
height: 400
num_classes: 5
